test/utils/createRecommendClient.ts

```diff
@@ -33,6 +33,10 @@ export const hit = {
   ],
 };
 
+export const initialState = {
+  recommendations: [hit],
+};
```

Comment:
This could be in a `test/fixtures` file, but this doesn't belong to the API response.